### PR TITLE
Code review request Prac 08 feedback

### PR DESCRIPTION
**Pull request by Sathvika-7284**
Pls provide me a code review,
@CynthiaJCU
@ArthurMY
@Nguyentananh98
@hjq0127

### CODE REVIEW

**Comment by tananhnguyen** [comment]
Nice work done!

File: dynamic_labels.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+class DynamicLabel(App):
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.names = ["apple", "pear", "banana"]
+
+    def build(self):
+        self.title = "Dynamic Labels"
```
Comment (tananhnguyen) [inline dynamic_labels.py:18]: lacking comment on functions

Comment (Sathvika-7284) [reply dynamic_labels.py:18]: Will add comments, thanks!

**Comment by ArthurMY** [comment]
Good job!

File: squaring.kv [hunk 2]
```diff
@@ -0,0 +1,19 @@
+BoxLayout:
```
Comment (ArthurMY) [inline squaring.kv:1]: nice work!

**Pull request merged**